src/ops/info.rs

```diff
@@ -50,14 +52,20 @@ pub fn info(
 
     validate_locked_and_frozen_options(package_id, config)?;
 
-    let current_rustc_version = &config.load_global_rustc(ws.as_ref())?.version;
+    let msrv_from_nearest_manifest_or_ws =
+        try_get_msrv_from_nearest_manifest_or_ws(config.cwd(), ws.as_ref());
+    // If the workspace does not have a specific Rust version,
+    // or if the command is not called within the workspace, then fallback to the global Rust version.
+    let rustc_version =
+        msrv_from_nearest_manifest_or_ws.unwrap_or(config.load_global_rustc(ws.as_ref())?.version);
```

Comment:
Shouldn't that be an `unwrap_or_else`?  rustc calls can be expensive, any spawn on Windows is expensive.

Reply:
Updated. Thanks!